advice/melissa-pignalosa-advice.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" href="/Users/melissapignalosa/Desktop/git-practice/css/melissa-pignalosa.css">
```

Comment:
Hey there! Let's make this a relative link so that your page can pull in the stylesheet regardless of where the project is hosting (i.e. - elsewhere than on your computer). :)